metricbeat/mb/parse/url.go

```diff
@@ -77,8 +78,25 @@ func (b URLHostParserBuilder) Build() mb.HostParser {
 		} else {
 			path = b.DefaultPath
 		}
+		// Normalize path
+		path = strings.Trim(path, "/")
```

Comment:
I'm never sure if we should try to be smart about / prefix and post fixes or just leave them.

Reply:
I added this because, without it, some existing tests were failing. I didn't want to change the tests because they might represent expectations of actual usage (i.e. the existing contract).